src/vcpkg/vcpkgpaths.cpp

```diff
@@ -325,6 +375,24 @@ namespace vcpkg
 
         Checks::check_exit(VCPKG_LINE_INFO, !root.empty(), "Error: Could not detect vcpkg-root.");
         Debug::print("Using vcpkg-root: ", root, '\n');
+        const auto vcpkg_root_file = root / ".vcpkg-root";
+        auto bundle_file = filesystem.read_contents(vcpkg_root_file, VCPKG_LINE_INFO);
```

Comment:
@ras0219-msft This caused https://github.com/microsoft/vcpkg/issues/21604

Reply:
On it, thanks for the notification!

Reply:
We've rolled back the release pending a solution.